app/controllers/application_controller.rb

```diff
@@ -10,17 +10,34 @@ class ApplicationController < ActionController::Base
   # Smartly redirect user back to URL they requested before authenticating
   # From: https://github.com/heartcombo/devise/wiki/How-To:-Redirect-back-to-current-page-after-sign-in,-sign-out,-sign-up,-update
   before_action :store_user_location!, if: :storable_location?
+  before_action :root_breadcrumb
```

Comment:
Does this wind up creating a dashboard breadcrumb even on the homepage?

Reply:
Yes, good point.  Let me sort that out -- this should probably move to the dashboard controller.

Reply:
Had to leave in application controller to ensure the dashboard link is rendered on all other pages, but guarded against showing on the home page